src/libs/README.md

```diff
@@ -0,0 +1,9 @@
+Jsoncons
```

Comment:
ADD:
Libraries used by Polycube
==================

This folder includes all the libraries that are required by Polycube and that are statically included in the source files.

Although this makes more difficult the update of the above libraries to more recent versions, these are used for limited purposes and hence this necessity should not be so common. In exchange, this guarantees that the Polycube code always compiles with the above components.

Reply:
Added :+1: thanks